src/suricata.c

```diff
@@ -2670,6 +2670,8 @@ int PostConfLoadedSetup(SCInstance *suri)
 
     MacSetRegisterFlowStorage();
 
+    SigTableInit();
```

Comment:
Oh, I bet the fuzzers require an update due to this.

Reply:
@catenacyber can you do a PR to fix this post-merge of the PR?

Reply:
Trivial patch: https://github.com/OISF/suricata/pull/12109/commits/1e13a3fa5eb5a7cbbc08fe5d1b59edd056863db0#diff-f327f4d38722d8a1e01f7ded5d72692b48b5a77d450f4995019e791af4182d07